spec/queries/matched_for_notification_spec.rb

```diff
@@ -29,6 +29,15 @@
                                                    {
                                                      all: ["uuid-567", "uuid-678"]
                                                    } })
+
+      @list8 = create(:subscriber_list, links: { topics:
```

Comment:
These lists could be renamed, e.g.
instead of `@list7` and `@list8`
could have `@list_of_all_topics_and_all_policies` and `@list_of_all_topics_and_any_policies`